controllers/v1/programs.js

```diff
@@ -4,11 +4,118 @@
  * created-date : 09-Mar-2022
  * Description : programs related information.
  */
+
+
+// Dependencies
+const programsHelper = require(MODULES_BASE_PATH + "/programs/helper")
+
+
 module.exports = class  Programs extends Abstract{
     constructor() {
       super("programs");
     }
+    
+
+   /**
+   * Create program.
+   * @method
+   * @name create
+   * @param {Object} req - requested data.
+   * @returns {JSON} - created program document.
+   */
+
+    async create(req) {
+      return new Promise(async (resolve, reject) => {
+        try {
   
+          req.body.userId = req.userDetails.userId;
+          let programCreationData = await programsHelper.create(
+            req.body,
+            true
+          );
+          
+          return resolve({
```

Comment:
@Prajwal17Tunerlabs pass this from the helper and directly resolve the programCreationData here